packages/dds/tree/src/feature-libraries/chunked-forest/codec/chunkDecoding.ts

```diff
@@ -234,6 +234,20 @@ export class InlineArrayDecoder implements ChunkDecoder {
 	}
 }
 
+/**
+ * Top-level decoder for incremental chunks.
+ * An encoded incremental chunk can contain multiple nodes. This decoder will decode all the nodes in a chunk.
+ */
+const incrementalRootDecoder: ChunkDecoder = {
```

Comment:
I do not think splitting this out into its own `ChunkDecoder` makes sense. Generally a chunk decoder reads some chunk of data described by a shape from a stream. This does not do that. There is no lonked/documented shape this decodes, and it reads the entire stream.

If you want this to exist as its own seperate thing (I'm unclear why you would want that unless you want to test it, but you don't as its not exported), I think a regular function would make more sense than a ChunkDecoder.

Reply:
I removed this. The tests were failing at some point on using `anyDecoder` and that is why I had added this. Looks like it's not needed though.